src/systems/filecoin_mining/storage_mining/storage_mining_subsystem.go

```diff
@@ -79,11 +79,16 @@ func (sms *StorageMiningSubsystem_I) _tryLeaderElection() {
 		return // fail to generate post candidates
 	}
 
+	// TODO Fix
+	var currState stateTree.StateTree
 	winningCandidates := make([]sector.PoStCandidate, 0)
+	st := sms._getStorageMinerActorState(currState, sms._keyStore().MinerAddress())
 
 	for _, candidate := range candidates {
-		// TODO align on worker address
-		if sms._consensus().IsWinningPartialTicket(candidate.PartialTicket()) {
+		// TODO: fix
```

Comment:
Ideally this type of TODO should be accompanied by a panic (or util.TODO call), since otherwise we may risk missing it and assuming the functionality is ready.

Reply:
sure will fix. THough we should check all TODOs

Reply:
@porcuquine have a sense of where I might get the SectorNumber given the PostVerifyInfo?